tests/AutoReview/FixerTest.php

```diff
@@ -74,23 +97,78 @@ public function testFixerDefinitions(FixerInterface $fixer)
                 $sample instanceof FileSpecificCodeSampleInterface ? $sample->getSplFileInfo() : $dummyFileInfo,
                 $tokens
             );
-            $this->assertTrue($tokens->isChanged(), sprintf('[%s] Sample #%d is not changed during fixing.', $fixer->getName(), $sampleCounter));
+            $this->assertTrue($tokens->isChanged(), sprintf('[%s] Sample #%d is not changed during fixing.', $fixerName, $sampleCounter));
 
             $duplicatedCodeSample = array_search(
                 $sample,
                 array_slice($samples, 0, $sampleCounter),
                 false
             );
+
             $this->assertFalse(
                 $duplicatedCodeSample,
-                sprintf('[%s] Code sample #%d duplicates #%d.', $fixer->getName(), $sampleCounter, $duplicatedCodeSample)
+                sprintf('[%s] Code sample #%d duplicates #%d.', $fixerName, $sampleCounter, $duplicatedCodeSample)
             );
         }
 
+        if ($fixerIsConfigurable) {
+            $options = $fixer->getConfigurationDefinition()->getOptions();
```

Comment:
I would remove the two `/** @var FixerOption $option */` comments below and add one here:
```php
/** @var FixerOption[] $options */
$options = $fixer->getConfigurationDefinition()->getOptions();
```